analyzers/rspec/cs/S5659_c#.html

```diff
@@ -1,9 +1,7 @@
 <p>If a JSON Web Token (JWT) is not signed with a strong cipher algorithm (or not signed at all) an attacker can forge it and impersonate user
 identities.</p>
-<ul>
-  <li> Don't use <code>none</code> algorithm to sign or verify the validity of an algorithm. </li>
-  <li> Don't use a token without verifying its signature before. </li>
-</ul>
+<p> * Don't use <code>none</code> algorithm to sign or verify the validity of a token.</p>
```

Comment:
please run the rspec script again for S5659 - there's a glitch in the rule-api when there is a ` *` in the JIRA instead of `*` - I removed the space from the RSPEC and rerunning it should correct it